style.css

```diff
@@ -286,3 +293,87 @@ h5 {
   padding-top: 36px;
   padding-bottom: 36px;
 }
+
+.contact-page {
+  display: flex;
+  flex-direction: column;
+  background-color: white;
+}
+
+.contact_items_container {
+  background-color: #6070ff;
+  background-image: url(/images/contact\ form\ shapes-mobile.svg);
+  background-size: contain;
```

Comment:
- [ ] Well done, you have used the correct background image as per the Figma design, but it is not showing and I know why, your path to the image is corrupted, to solve this, please just remove the `/` before images and it will be good. 
![bongo](https://user-images.githubusercontent.com/77843050/206165750-eeb53d28-1942-4450-9f54-5aa45cd68196.gif)

- [ ] One more thing, take a look back at the Figma design, at the bottom of the page, there is a black line which is not present in your design, kindly add that and you will be good to go

Reply:
@joskalenda Thank you for the feedback.